scripts/qa_crowbarsetup.sh

```diff
@@ -3604,6 +3605,42 @@ function oncontroller_run_tempest()
     return $tempestret
 }
 
+function oncontroller_run_integration_test()
+{
+    # Install Mozilla Firefox < 47 to work with Selenium.
+    safely zypper -n in 'MozillaFirefox<47'
+
+    # Add Devel:Languages:Python repo (no GPG checks) to install Selenium
+    local dlp="http://download.opensuse.org/repositories/devel:/languages:/python/$slesdist/"
+    zypper ar --no-gpgcheck --refresh $dpl python
+    safely zypper -n in python-selenium
+    safely zypper -n in python-nose
+    safely zypper -n in python-xvfbwrapper
+
+    # Configure the tests to be headless
+    export WITH_SELENIUM=1
+    export SELENIUM_HEADLESS=1
+    export INTEGRATION_TESTS=1
+
+    cd /srv/www/openstack-dashboard/openstack_dashboard/test/integration_tests
```

Comment:
Or maybe we dont even need to mv into that dir, instead telling crudini to do the changes to the $basepath/horizon.conf file instead?